gprofiler/utils/__init__.py

```diff
@@ -13,64 +13,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import ctypes
-import datetime
-import glob
-import logging
 import os
-import random
 import re
-import shutil
-import signal
-import socket
-import string
-import subprocess
-import sys
-import time
-from contextlib import contextmanager
 from functools import lru_cache
-from pathlib import Path
-from subprocess import CompletedProcess, Popen, TimeoutExpired
-from tempfile import TemporaryDirectory
-from threading import Event
-from typing import Any, Callable, Dict, Iterator, List, Optional, Tuple, Union, cast
+from typing import List
 
 import importlib_resources
 import psutil
-from granulate_utils.exceptions import CouldNotAcquireMutex
-from granulate_utils.linux.mutex import try_acquire_mutex
-from granulate_utils.linux.ns import run_in_ns
 from granulate_utils.linux.process import is_kernel_thread, process_exe
 from psutil import Process
 
-from gprofiler.consts import CPU_PROFILING_MODE
-from gprofiler.platform import is_linux, is_windows
+from granulate_utils.gprofiler.platform import is_windows
+import granulate_utils.gprofiler.utils as _utils
```

Comment:
Note that moving `run_process` to granulate-utils poses a problem, since in another PR we're adding _more_ gprofiler-specific behavior to that function.